components/shared/infoBox/InfoBoxItem.tsx

```diff
@@ -0,0 +1,42 @@
+import { useRouter } from "next/navigation"
+import React from "react"
+
+import { Button } from "@/components/shared/button/Button"
+import useSearchMachineActor from "@/lib/machines/search/useSearchMachineActor"
+
+type InfoBoxItemProps = {
+  term: string
+  description: string[]
+  isButtons?: boolean
+}
+
+const InfoBoxItem = ({ term, description, isButtons = false }: InfoBoxItemProps) => {
+  const router = useRouter()
+  const actor = useSearchMachineActor()
+
+  const handleClick = (text: string) => {
+    actor.send({ type: "SEARCH" })
```

Comment:
We should not do anything with the actor here I think. The search result page should take whatever is in the serachparams

Reply:
yeah that doesn't really work. I'll need Mikkel to help with that. But I can remove this actor line 😉

Reply:
I have made some changes to the work result page that should re-instantiate the search machine when landing on the page. Have you tied pulling in main and se if its still necessary?

Reply:
I deleted this line as you suggested. And yes, I pulled throughout the process :) Haven't tested whether it works on the search result page though.